sandbox/functions-recipes/durable-functions.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Azure Functions Recipes - Durable Functions
+description: Description
+author: cecilphillip
+manager: scottca
+keywords:
+ms.topic: article
+ms.date: 04/17/2018
+ms.author: cephilli
+ms.devlang: csharp
+ms.service: functions
+---
+
+# Durable Functions
+
+[!include[](../includes/header.md)]
+
+| | |
+|---|---|
+|<h2>Setting up Orchestrations</h2> | |
+[Starting Orchestrations](durable-setup.md#starting-orchestrations) | Start a stateful workflow
```

Comment:
We typically use the term "orchestration" rather than "workflow". The LogicApps folks prefer this as well.